### PR TITLE
feat(issue_details): Implement new UI for loaded images

### PR DESCRIPTION
**Pull request by jan-auer**
Implements a new UI for "Loaded Images" in issue details. It is a visual combination of our Breadcrumbs section and the settings UI for uploaded debug files:

- Allows to search by file names, identifiers, and also by address (performs a range lookup)
- By default, only referenced images are shown. Optionally, all can be toggled
- By default, only a summary of the symbolication status is shown. Optionally, all available info can be displayed. The legacy fallback (where this information is not available) shows the debug id in place.
- Shows the code file name and optionally the debug file name if available.
- Displays available debug information (such as, whether sources have been available).
- Instead of the collapsed block (which cannot be recollapsed), this uses a scrollable list.

debug-meta

### CODE REVIEW

**Comment by jan-auer** [comment]
Symbolicator integration tests will be fixed with the next symbolicatore release.

**Comment by jan-auer** [comment]
One update to the GIF: The icons in the status line are now gone. This was removed to avoid creating a new UI pattern:

image

**Comment by jan-auer** [comment]
This also fixes Symbolicator failures that will be happening on `master` and other PRs.

**Review by markstory — COMMENTED** [review]
We've generally been requesting that new components be created as typescript files. If you're planning on iterating on this further could that be taken care of then?

**Review by markstory — APPROVED** [review]

**Comment by jan-auer** [comment]
> We've generally been requesting that new components be created as typescript files.

Yes, we'll have UI engineers joining soon that will be looking at this. If possible, I'd like to avoid refactoring this component in TS :)

File: src/sentry/static/sentry/app/components/events/eventEntries.jsx [hunk 1]
```diff
@@ -130,7 +130,8 @@ class EventEntries extends React.Component {
         return (
           <Component
             key={'entry-' + entryIdx}
-            projectId={project.slug}
+            projectId={project ? project.slug : null}
+            orgId={organization ? organization.slug : null}
```
[the remark this answers is not in the document]
Comment (jan-auer) [reply src/sentry/static/sentry/app/components/events/eventEntries.jsx]: @markstory Updated, could you please double-check that I found the right spots?

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Comment by dcramer** [comment]
We need to get this symbolicator fix in right now, and ideally without adding a bunch of new features that will break when the PR owners are asleep.

**Comment by jan-auer** [comment]
Merging now, after approving Percy.

**Pull request merged**